package.json

```diff
@@ -43,8 +43,8 @@
     "xmldom": "0.1.27"
   },
   "peerDependencies": {
-    "react": ">= 16.2.0",
-    "react-native": " >= 0.52.2",
+    "react": ">= 16.6.0",
```

Comment:
The context API should work with `>= 16.3.0` as well

Reply:
The `contextType` syntax was added in 16.6.0. Since all of the docs refer to it, I thought it was better to use a min version that matches the docs.